locale/de_DE/api.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/de_DE/api.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
+  * Copyright (c) 2000-2017 John Willinsky
+  * Copyright (c) 2017 Stefan Weil, UB Mannheim
+  * Distributed under the GNU GPL v2.
```

Comment:
please leave the header as it is. the file is located in the applications path at the given location - it's relative to the app's root, not to .../lib/pkp/...